### PR TITLE
[ECP-8686] Fix webhook E2E test

### PR DESCRIPTION
**Pull request by hossam-adyen**
**Description**
Fix  E2E related Webhook tests

### CODE REVIEW

File: .github/Makefile [hunk 1]
```diff
@@ -48,6 +48,8 @@ configure: n98-magerun2.phar
 
 	./n98-magerun2.phar config:store:set --encrypt payment/adyen_abstract/api_key_test "${ADYEN_API_KEY}" > /dev/null
 	bin/magento config:set payment/adyen_abstract/client_key_test "${ADYEN_CLIENT_KEY}"
+	bin/magento config:set payment/adyen_abstract/notification_username "${WEBHOOK_USERNAME}"
+	bin/magento config:set --encrypt payment/adyen_abstract/notification_password "${WEBHOOK_PASSWORD}"
```
Comment (hossam-adyen) [inline .github/Makefile:52]:
```suggestion
	./n98-magerun2.phar config:store:set --encrypt payment/adyen_abstract/notification_password "${WEBHOOK_PASSWORD}"
```

**Comment by hossam-adyen** [comment]
Finally it's working

https://github.com/Adyen/adyen-magento2/actions/runs/7143524995

File: .github/workflows/templates/docker-compose.yml [hunk 2]
```diff
@@ -54,6 +54,7 @@ services:
       - composer:/usr/local/bin
       - magento:/var/www/html
       - ../../Makefile:/var/www/html/Makefile
+      - ../../../magento-logs:/var/www/html/var/log
```
Comment (hossam-adyen) [inline .github/workflows/templates/docker-compose.yml:57]:
```suggestion
      # - ../../../magento-logs:/var/www/html/var/log
```

File: .github/workflows/e2e-test-dispatch.yml [hunk 3]
```diff
@@ -98,12 +100,21 @@ jobs:
           ADYEN_MERCHANT: ${{secrets.ADYEN_MERCHANT}}
           GOOGLE_USERNAME: ${{secrets.GOOGLE_USERNAME}}
           GOOGLE_PASSWORD: ${{secrets.GOOGLE_PASSWORD}}
-          WEBHOOK_USERNAME: ${{secrets.WEBHOOK_USERNAME}}
-          WEBHOOK_PASSWORD: ${{secrets.WEBHOOK_PASSWORD}}
+          WEBHOOK_USERNAME: admin
+          WEBHOOK_PASSWORD: 1234
+      #          WEBHOOK_USERNAME: ${{secrets.WEBHOOK_USERNAME}}
+      #          WEBHOOK_PASSWORD: ${{secrets.WEBHOOK_PASSWORD}}
```
Comment (hossam-adyen) [inline .github/workflows/e2e-test-dispatch.yml:106]:
```suggestion
      WEBHOOK_USERNAME: ${{secrets.WEBHOOK_USERNAME}}
      WEBHOOK_PASSWORD: ${{secrets.WEBHOOK_PASSWORD}}
```

File: .github/Makefile [hunk 4]
```diff
@@ -41,13 +41,16 @@ configure: n98-magerun2.phar
...
-	bin/magento config:set payment/adyen_abstract/capture_mode 'auto'
+	bin/magento config:set payment/adyen_abstract/capture_mode 'manual'
 	bin/magento config:set payment/adyen_abstract/paypal_capture_mode 0
 	bin/magento config:set payment/adyen_abstract/recurring_configuration '{"adyen_cc":{"name":"Credit Card","enabled":"1","recurringProcessingModel":"CardOnFile"}}'
 	bin/magento config:set payment/adyen_cc_vault/require_cvc 1
-
-	./n98-magerun2.phar config:store:set --encrypt payment/adyen_abstract/api_key_test "${ADYEN_API_KEY}" > /dev/null
 	bin/magento config:set payment/adyen_abstract/client_key_test "${ADYEN_CLIENT_KEY}"
+	bin/magento config:set payment/adyen_abstract/notification_username 'admin'
+	./n98-magerun2.phar config:store:set --encrypt payment/adyen_abstract/notification_password '1234'  > /dev/null
```
Comment (hossam-adyen) [inline .github/Makefile:50]:
```suggestion
	bin/magento config:set payment/adyen_abstract/notification_username '${WEBHOOK_USERNAME}'
	./n98-magerun2.phar config:store:set --encrypt payment/adyen_abstract/notification_password '${WEBHOOK_PASSWORD}'  > /dev/null
```

File: .github/Makefile [hunk 5]
```diff
@@ -41,13 +41,16 @@ configure: n98-magerun2.phar
...
+	./n98-magerun2.phar config:store:set --encrypt payment/adyen_abstract/api_key_test "${ADYEN_API_KEY}" > /dev/null
+#	bin/magento config:set payment/adyen_abstract/notification_username "${WEBHOOK_USERNAME}"
+#	./n98-magerun2.phar config:store:set --encrypt payment/adyen_abstract/notification_password "${WEBHOOK_PASSWORD}"
```
Comment (hossam-adyen) [inline .github/Makefile:53]:
```suggestion
```

File: .github/workflows/e2e-test-dispatch.yml [hunk 6]
```diff
@@ -98,12 +100,19 @@ jobs:
           ADYEN_MERCHANT: ${{secrets.ADYEN_MERCHANT}}
           GOOGLE_USERNAME: ${{secrets.GOOGLE_USERNAME}}
           GOOGLE_PASSWORD: ${{secrets.GOOGLE_PASSWORD}}
-          WEBHOOK_USERNAME: ${{secrets.WEBHOOK_USERNAME}}
-          WEBHOOK_PASSWORD: ${{secrets.WEBHOOK_PASSWORD}}
+      WEBHOOK_USERNAME: ${{secrets.WEBHOOK_USERNAME}}
+      WEBHOOK_PASSWORD: ${{secrets.WEBHOOK_PASSWORD}}
```
Comment (hossam-adyen) [inline .github/workflows/e2e-test-dispatch.yml:104]:
```suggestion
           WEBHOOK_USERNAME: ${{secrets.WEBHOOK_USERNAME}}
           WEBHOOK_PASSWORD: ${{secrets.WEBHOOK_PASSWORD}}
```

File: .github/scripts/e2e.sh [hunk 7]
```diff
@@ -21,6 +21,10 @@ case $option in
         echo "Running Express Checkout E2E Tests."
         npm run test:ci:magento:express-checkout
         ;;
+    "webhook")
```
Comment (hossam-adyen) [inline .github/scripts/e2e.sh:24]:
```suggestion
    "backoffice")
```

File: .github/workflows/e2e-test-dispatch.yml [hunk 8]
```diff
@@ -12,6 +12,10 @@ on:
         description: "Express Checkout Repository Pipeline"
         required: true
         default: "develop"
+      testGroup:
+        description: "Test group all|express-checkout|backoffice"
+        required: true
+        default: "all"
```
Comment (hossam-adyen) [inline .github/workflows/e2e-test-dispatch.yml:18]:
```suggestion
      testGroup:
        description: "Test group"
        required: true
        default: "all"
        type: choice
        options:
        - all
        - standard
        - express-checkout
```

File: .github/scripts/e2e.sh [hunk 9]
```diff
@@ -21,6 +21,10 @@ case $option in
         echo "Running Express Checkout E2E Tests."
         npm run test:ci:magento:express-checkout
         ;;
+    "backoffice")
+        echo "Running Webhook E2E Tests."
+        npm run test:ci:magento:webhook
+        ;;
```
Comment (hossam-adyen) [inline .github/scripts/e2e.sh:27]:
```suggestion
```
```suggestion
    "backoffice")
        echo "Running Webhook E2E Tests."
        npm run test:ci:magento:webhook
        ;;
```

**Review by RokPopov — APPROVED** [review]

**Pull request merged**